iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -585,6 +584,163 @@ TEST_F(PortManager_test, DoDiscoveryPublisherCanWaitAndSubscriberDiscardOldestLe
     EXPECT_THAT(subscriber.getSubscriptionState(), Eq(iox::SubscribeState::SUBSCRIBED));
 }
 
+TEST_F(PortManager_test, SubscriberRequiringHistoryDoesNotConnectToPublisherWithoutHistorySupport)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "43f3ea1e-777a-4cc6-8478-13f981b7c941");
+
+    constexpr uint64_t historyRequest = 1;
```

Comment:
Note that I would like to introduce the coding convention that local constants (like here) should not be all caps. I find the flow to read them like this much more pleasant. As an added bonus it is easier to distinguish them from "configuration constants" with a much a larger scope.

Those constants here mainly serve to express intent and make the tests easier to understand (they are the essential part of the test configuration).

Reply:
Hmm, not sure about this. We already use a different coding convention all over the place and it would be another exception to remember.

Reply:
Fine by me but I would bet that we have to change it to all caps when we want to certify the code because then we have to blindly follow the rules. I have no idea if we can make them so fine grained.

Reply:
I think we can define coding rules like this for certification, whether we want to is another issue. There will be different tool settings needed for tests and library code for sure, since e.g. we use e.g. dynamic memory in tests for simplicity etc. From this point of view we could allow it for tests without a problem I think.

I kind of dislike to make the code uglier than it needs to be for this purpose only, and overusing ALL_CAPS_SNAKE_CASE is annoying to read.